windshaft/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:vivid
```

Comment:
Fixed a changed dependency and added `build-essential` to get install to complete, but the resulting Windshaft build is broken and will not start. Going to stick with Ubuntu image.